### PR TITLE
Various enhancements

### PR DESCRIPTION
**Pull request by ToastyStoemp**

### CODE REVIEW

File: src/commands/afk.js [hunk 1]
```diff
@@ -4,13 +4,17 @@ var init = function(bot)
 
 	bot.on("chat", function(data)
 	{
-		for(var i = 0; i < bot.afks.length; i++)
+		if(data.nick !== bot.nick)
 		{
-			var name = "@" + bot.afks[i];
-			if(data.text.indexof(name) !== -1)
-				bot.send(name + " is afk!");
+			console.log(bot.nick);
```
Comment (M4GNV5) [inline src/commands/afk.js]: debugging lines in the pr?

File: src/commands/help.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ var help = function(bot, sender, args)
 		if(requiredLvl <= lvl)
 			cmds.push(key);
 	}
-
+	cmds = cmds.sort();
```
Comment (M4GNV5) [inline src/commands/help.js:13]: what is this for?

File: src/commands/afk.js (hunk 1 above)
Comment (ToastyStoemp) [inline src/commands/afk.js]: oh forgot to delete

File: src/commands/help.js (hunk 2 above)
Comment (ToastyStoemp) [inline src/commands/help.js:13]: sorting the cmds

Comment (M4GNV5) [inline src/commands/help.js:13]: ahh i see its in the help.js ok

**Pull request merged**